src/generic.rs

```diff
@@ -6,6 +6,7 @@
 
 use alloc::vec::Vec;
 
+#[cfg(feature = "persistance")]
 use arbitrary::Arbitrary;
```

Comment:
Add a new line after this (help to isolate visually the module under feature flag)